linux_os/guide/system/permissions/partitions/mount_option_tmp_noexec/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel6,rhel7,rhel8,sle15
```

Comment:
Why is sle15 removed?

Reply:
It might be part of the merging and rebasing issues.
It was not my intention to remove it.
I will restore it.